### PR TITLE
Toyota: Prius V fixups

### PR DESCRIPTION
**Pull request by CT921**
Thank you Comma team for merging the PR of Prius V (Prius Alpha) car port.
But in the three-month period between submitting the PR and being merged.
During this period, I installed SDSU to confirm that Prius V fully supports SNG. And I optimized the control parameter.
Hope Comma can merge the following commits.

TestRoute("a2adabc3165216a5|2022-01-25--21-18-16", TOYOTA.PRIUS_V),

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -1664,7 +1664,7 @@ class CAR:
   CAR.RAV4H: dbc_dict('toyota_tssp_hybrid_generated', 'toyota_adas'),
   CAR.RAV4: dbc_dict('toyota_rav4_2017_pt_generated', 'toyota_adas'),
   CAR.PRIUS: dbc_dict('toyota_prius_2017_pt_generated', 'toyota_adas'),
-  CAR.PRIUS_V: dbc_dict('toyota_rav4_2017_pt_generated', 'toyota_adas'),
+  CAR.PRIUS_V: dbc_dict('lexus_ct200h_2018_pt_generated', 'toyota_adas'),
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:1667]: these are the same

Comment (CT921) [reply selfdrive/car/toyota/values.py]: Updated

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -1714,4 +1714,4 @@ class CAR:
 NO_DSU_CAR = TSS2_CAR | {CAR.CHR, CAR.CHRH, CAR.CAMRY, CAR.CAMRYH}
 
 # no resume button press required
-NO_STOP_TIMER_CAR = TSS2_CAR | {CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.SIENNA, CAR.LEXUS_ESH}
+NO_STOP_TIMER_CAR = TSS2_CAR | {CAR.PRIUS_V, CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.SIENNA, CAR.LEXUS_ESH}
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:1717]: are you sure this is right? you don't have to press a button to start going after entering a standstill?

File: selfdrive/car/toyota/interface.py [hunk 3]
```diff
@@ -39,11 +39,12 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.steerActuatorDelay = 0.3
 
     elif candidate == CAR.PRIUS_V:
+      stop_and_go = True
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/interface.py:42]: does stock ACC have stop and go? if not, this should stay as it. shouldn't affect your smartDSU

**Comment by adeebshihadeh** [comment]
Can you rebase? There's a conflict

File: selfdrive/car/toyota/interface.py (hunk 3 above)
Comment (CT921) [reply selfdrive/car/toyota/interface.py:42]: Stock ACC have stop and go in worldwide. But Toyota block the function in Taiwan.

File: selfdrive/car/toyota/values.py (hunk 2 above)
Comment (CT921) [reply selfdrive/car/toyota/values.py:1717]: I've confirmed that you can check out this TestRoute. 
TestRoute("a2adabc3165216a5|2022-01-25--21-18-16", TOYOTA.PRIUS_V),Route

**Comment by cydia2020** [comment]
> During this period, I installed SDSU to confirm that Prius V fully supports SNG

Should also update the min ACC speed in cars.md

**Pull request merged**